x-pack/plugin/enrich/src/main/java/org/elasticsearch/xpack/enrich/action/EnrichShardMultiSearchAction.java

```diff
@@ -315,7 +315,7 @@ private static SearchResponse createSearchResponse(TopDocs topDocs, SearchHit[]
             1,
             1,
             0,
-            1L,
+            1L, // TODO: This does not measure "took", as it's part of an internal API. What should we do here?
```

Comment:
@javanna I'm guessing that this does not require instrumentation as "took" isn't even recorded here, correct?

Reply:
correct.